layers/+tools/transmission/README.org

```diff
@@ -63,52 +64,68 @@ Add =(transmission :variables transmission-auto-refresh-all t)= to
 
 ** =transmission-mode=
 
-| Key binding | Description                                                   |
-|-------------+---------------------------------------------------------------|
-| ~SPC m g t~ | Open a `transmission-files-mode' buffer for torrent at point. |
-| ~SPC m a t~ | Add torrent by filename, URL, magnet link, or info hash.      |
-| ~SPC m s d~ | Set global download speed limit in kB/s.                      |
-| ~SPC m g p~ | Open a =transmission-peers-mode= buffer for torrent at point. |
-| ~SPC m g i~ | Open a =transmission-info-mode= buffer for torrent at point.  |
-| ~SPC m s l~ | Set global seed ratio limit.                                  |
-| ~SPC m m~   | Move torrent at point or in region to a new location.         |
-| ~SPC m r~   | Prompt to remove torrent at point or torrents in region.      |
-| ~SPC m t s~ | Toggle torrent between started and stopped.                   |
-| ~SPC m a a~ | Add announce URLs to torrent or torrents.                     |
-| ~SPC m s u~ | Set global upload speed limit in kB/s.                        |
-| ~SPC m v~   | Verify torrent at point or in region.                         |
-| ~SPC m q~   | Quit and bury the buffer.                                     |
-| ~SPC m s b~ | Set bandwidth priority of torrent(s) at point or in region.   |
+| Key binding | Description                                                       |
+|-------------+-------------------------------------------------------------------|
+| ~H~         | Move torrent to the top of the queue.                             |
+| ~J~         | Move torrent down the queue by one.                               |
+| ~K~         | Move torrent up the queue by one.                                 |
+| ~L~         | Move torrent to the bottom of the queue.                          |
+| ~SPC m m~   | Move torrent at point or in region to a new location.             |
+| ~SPC m q~   | Quit and bury the buffer.                                         |
+| ~SPC m v~   | Verify torrent at point or in region.                             |
+| ~SPC m D~   | Prompt to remove and delete torrent at poit or torrens in region. |
+| ~SPC m R~   | Prompt to remove torrent at point or torrents in region.          |
+| ~SPC m S~   | Sort torrents according to the column at point.                   |
+| ~SPC m a a~ | Add torrent by filename, URL, magnet link, or info hash.          |
```

Comment:
above each prefix you can add a heading, for example
```txt
| ~SPC m S~   | Sort torrents according to the column at point.                   |
|-------------+-------------------------------------------------------------------|
| Add         |                                                                   |
|-------------+-------------------------------------------------------------------|
| ~SPC m a a~ | Add torrent by filename, URL, magnet link, or info hash.          |
```

Reply:
Adding headers is an easy task as well. It'll make reading the README much smoother, got it.

I'll remove `transmission-trackers-add` from `transmission-mode-map` and just leave it in `transmission-info-mode-map` where trackers are listed explicitly.

Reply:
After checking on the documentation for `transmission-trackers-add`, I believe that it should remain in `transmission-mode-map` because it should be used to add trackers to the selected torrent, or multiple marked torrents even in `transmission-mode`. For that reason, we should leave `SPC m a` as a prefix for `SPC m a a` and `SPC m a t`.